JDI.Light/JDI.Light/Elements/Composite/MultiDropdown.cs

```diff
@@ -0,0 +1,138 @@
+﻿using JDI.Light.Elements.Base;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using JDI.Light.Attributes;
+using JDI.Light.Elements.Common;
+
+namespace JDI.Light.Elements.Composite
+{
+    public class MultiDropdown : UIElement
+    {
+        [FindBy(Tag = "button")]
+        private Button _header;
```

Comment:
it would be nice to be able to set up the value for _header via constructor
![image](https://user-images.githubusercontent.com/6689884/54746847-49884100-4bde-11e9-895b-be4a782895ef.png)

Reply:
It set up automatically, using basic searching elements mechanism